docs/faq.md

```diff
@@ -73,6 +73,15 @@ Since the Baseline is designed for the developers of a project, not the consumer
 OSPS Baseline compliance is a point-in-time status.
 We encourage projects using the OSPS Baseline to say something like “As of April 31, 2025, this project complies with OSPS Baseline version 2025-02-30 level 2.”
 
+## What is in scope for OSPS Baseline?
+
+OSPS Baseline seeks to address security hygiene elements — those which secure the ways of working, delivering the product, and equipping its users to adopt it safely.
+
+To use an analogy, Baseline is similar to a food processing plant requiring practices to ensure safe food products: wearing hair nets, regularly cleaning and sanitizing equipment, monitoring refrigerator temperatures, etc.
```

Comment:
We may also want to add a mention of planning for reactive activity, such as safety recalls (we have controls for things like receiving and managing security notifications).